test/CMakeLists.txt

```
@@ -446,6 +446,17 @@ if (TARGET TBB::tbb)
     tbb_add_test(SUBDIR tbb NAME test_eh_thread DEPENDENCIES TBB::tbb)
     tbb_add_test(SUBDIR tbb NAME test_global_control DEPENDENCIES TBB::tbb)
     tbb_add_test(SUBDIR tbb NAME test_task DEPENDENCIES TBB::tbb)
+
+    if (TBB_FUZZ_TESTING AND NOT WIN32)
+        if (NOT ((CMAKE_CXX_COMPILER_ID STREQUAL Clang) OR (CMAKE_CXX_COMPILER_ID STREQUAL IntelLLVM)))
+            message(FATAL_ERROR "Fuzzing requires Clang or IntelLLVM compiler.")
+        endif()
+        tbb_add_test(SUBDIR tbb NAME test_fuzzing)
+        add_dependencies(test_fuzzing test_task)
+        target_compile_definitions(test_fuzzing PRIVATE CMD="$<TARGET_FILE:test_task> >/dev/null 2>&1")
```

```suggestion
        target_compile_definitions(test_fuzzing PRIVATE CMD="$<TARGET_FILE:test_task> > /dev/null 2>&1")
```
If we redirect to /dev/null/, I think we need to add one more check, that this is Unix.